### PR TITLE
private projects

### PR DESCRIPTION
**Pull request by n0n1m**
this PR offers a way to make a private project, that does not belong to honey team to your member page (it won't appear on page with all projects

There is [an example](https://github.com/honey-team/honey-team.github.io/compare/main...n0n1m:honey-test:test?expand=1#diff-b9a7a9907d21442f361c3623f70bb251812e29ebb691136c4fdaece1067b5b7a) of private project in htconfig

### CODE REVIEW

File: src/components/ProjectCard/ProjectCard.jsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ function ProjectCard({ project }, modificator) {
           {project.description}
         </div>
         <div className={styles["project-card__button"]}>
-          <a href={`https://github.com/honey-team/${project.gh}`}>
+          <a href={!project.gh.includes("/") ? `https://github.com/honey-team/${project.gh}` : `https://github.com/${project.gh}`}>
```
Comment (bleudev) [inline src/components/ProjectCard/ProjectCard.jsx]:
```suggestion
          <a href={project.gh.includes("/") ? `https://github.com/${project.gh}` : `https://github.com/honey-team/${project.gh}`}>
```

**Review by bleudev — CHANGES_REQUESTED** [review]
Can you optimise the ternary operator?

File: src/components/ProjectCard/ProjectCard.jsx (hunk 1 above)
Comment (n0n1m) [reply src/components/ProjectCard/ProjectCard.jsx]: Done

**Review by bleudev — APPROVED** [review]
Approved! Thanks for your work! 🎉

**Pull request merged**